### PR TITLE
Add cloudformation sample and shortcodes

### PR DESCRIPTION
**Pull request by brentley**
*Issue #, if available:*

Add cloudformation sample and shortcodes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by brentley** [comment]
example link: https://eksworkshop.com/pr-43/example_cf_templates/

File: content/example_cf_templates.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Example of using CF Templates"
```
Comment (dalbhanj) [inline content/example_cf_templates.md]: "Example of using CloudFormation Templates"

File: content/example_cf_templates.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+#### Click below to add a CloudFormation Stack
```
Comment (dalbhanj) [inline content/example_cf_templates.md]: Does this mean, contributors can click on the template to use it or should you change this to --> "#### Use the template below for CloudFormation Stack"

Comment (brentley) [reply content/example_cf_templates.md]: It's click to use

**Pull request merged**